src/main/java/io/anserini/reproduce/RunMsMarco.java

```diff
@@ -39,66 +33,81 @@ public class RunMsMarco {
 
   private static final String FAIL = RED + "[FAIL]" + RESET;
 
-  private static final String COLLECTION = "msmarco-v1-passage";
- 
+  private static String COLLECTION = "msmarco-v1-passage";
+
   public static void main(String[] args) throws Exception {
-    final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
-    try (InputStream inputStream = RunMsMarco.class.getClassLoader()
-        .getResourceAsStream("reproduce/msmarco-v1-passage.yaml")) {
-      if (inputStream == null) {
-        throw new FileNotFoundException("Resource file not found. Please check the file path.");
+
+    // check for cmd option
+    if (args.length >= 2 && "-v".equals(args[0])) {
+      String collectionVersion = args[1];
+      switch (collectionVersion) {
+          case "1":
+              COLLECTION = "msmarco-v1-passage";
+              break;
+          case "2":
+              COLLECTION = "msmarco-v2.1-doc";
```

Comment:
Ultimately, we want something like https://castorini.github.io/pyserini/2cr/msmarco-v2-doc.html

So it's okay with put doc and doc segmented together.